### ISSUE TITLE
ValueError at / invalid literal for int() with base 10: ''

### ISSUE THREAD

**Issue by shuboy2014** [getsentry/raven-python#1099]
I configure raven  with Django as per the documentation by sentry [here](https://docs.sentry.io/clients/python/integrations/django/) but when i run server i am getting this error

```
Internal Server Error: /
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/exception.py", line 39, in inner
    response = get_response(request)
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 244, in _legacy_get_response
    response = middleware_method(request)
  File "/usr/local/lib/python2.7/dist-packages/raven/contrib/django/middleware/__init__.py", line 135, in process_request
    request.body  # forces stream to be read into memory
  File "/usr/local/lib/python2.7/dist-packages/django/http/request.py", line 267, in body
    int(self.META.get('CONTENT_LENGTH', 0)) > settings.DATA_UPLOAD_MAX_MEMORY_SIZE):
ValueError: invalid literal for int() with base 10: ''
[02/Oct/2017 12:25:02] "GET / HTTP/1.1" 500 67787
```

**Comment by ashwoods** [comment]
Could you supply django version as well please.

**Comment by shuboy2014** [comment]
Django 1.10

**Comment by ashwoods** [comment]
This is a Django bug that was fixed here: https://github.com/django/django/commit/5c63b3e5a797102d915e1683971517f747a28013
and at least doesn't trip with django 1.10.8 

I'll catch the exception, but you might consider upgrading to the latest 1.10 version if possible.

**Comment by iivri** [comment]
I am getting that error in Django 1.11.7

**Comment by AlkhatibHamad** [comment]
> This is a Django bug that was fixed here: [django/django@5c63b3e](https://github.com/django/django/commit/5c63b3e5a797102d915e1683971517f747a28013)
> and at least doesn't trip with django 1.10.8
> 
> I'll catch the exception, but you might consider upgrading to the latest 1.10 version if possible.

This works for me so well, Thanks :heart:
